EDScoutWebUI/templates/index.html

```diff
@@ -257,11 +260,12 @@
                 return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
             }
 
-            function createFlagElement(iconName, activeIndicator) {
+            function createFlagElement(iconName, description, activeIndicator) {
                 const flagElementType = 'b';
-                let element = createIconElement(iconName, flagElementType);
+                let element = createIconElement(iconName, description, flagElementType);
                 if (!activeIndicator) {
                     element.classList.add("inactive")
+                    element.removeAttribute("title")

```

Comment:
Any need to remove the title here (I don't see it being re-added in the else below)?

Reply:
It's removed because the icon is always there but sometimes is styled as 'inactive' to hide it. Hiding it with CSS doesn't prevent the tooltip from showing, however, so the attribute is removed here to prevent it from popping up when hovering over 'inactive' flag elements.

Reply:
Yeah okay, that makes sense.